changelog/0.4.0/toolkit-test_0.4.0_2.13_changelog.md

```diff
@@ -33,24 +36,27 @@
 
  - org.scala-lang:toolkit-test_2.13:0.4.0
    - org.scala-lang:toolkit_2.13:0.4.0
-     - com.lihaoyi:os-lib_2.13:0.10.0
+     - com.lihaoyi:os-lib_2.13:0.10.2
        - com.lihaoyi:geny_2.13:1.1.0
-     - com.lihaoyi:upickle_2.13:3.3.0
-       - com.lihaoyi:ujson_2.13:3.3.0
-         - com.lihaoyi:upickle-core_2.13:3.3.0
+     - com.lihaoyi:upickle_2.13:3.3.1
+       - com.lihaoyi:ujson_2.13:3.3.1
+         - com.lihaoyi:upickle-core_2.13:3.3.1
            - com.lihaoyi:geny_2.13:1.1.0 (already listed)
-       - com.lihaoyi:upack_2.13:3.3.0
-         - com.lihaoyi:upickle-core_2.13:3.3.0 (already listed)
-       - com.lihaoyi:upickle-implicits_2.13:3.3.0
-         - com.lihaoyi:upickle-core_2.13:3.3.0 (already listed)
+       - com.lihaoyi:upack_2.13:3.3.1
+         - com.lihaoyi:upickle-core_2.13:3.3.1 (already listed)
+       - com.lihaoyi:upickle-implicits_2.13:3.3.1
+         - com.lihaoyi:upickle-core_2.13:3.3.1 (already listed)
      - com.softwaremill.sttp.client4:core_2.13:4.0.0-M14
        - com.softwaremill.sttp.model:core_2.13:1.7.10
        - com.softwaremill.sttp.shared:core_2.13:1.3.18
        - com.softwaremill.sttp.shared:ws_2.13:1.3.18
          - com.softwaremill.sttp.model:core_2.13:1.7.10 (already listed)
          - com.softwaremill.sttp.shared:core_2.13:1.3.18 (already listed)
      - com.softwaremill.sttp.client4:upickle_2.13:4.0.0-M14
-       - com.lihaoyi:upickle_2.13:3.3.0 (already listed)
+       - com.lihaoyi:upickle_2.13:3.3.0
```

Comment:
if we update sttp to `4.0.0-M16` then it will come with upickle 3.3.1

Reply:
or alternatively `//> using dep com.softwaremill.sttp.client4::upickle::4.0.0-M14,exclude=com.lihaoyi%%upickle` - but probably what I said above is probably better